editor/src/clj/editor/editor_extensions/ui_components.clj

```diff
@@ -218,50 +215,40 @@
+                                   (if maybe-column
+                                     (let [{:keys [grow]} maybe-column]
+                                       (cond-> ret grow (assoc grow-key :always)))
+                                     ret)))
+                               constraints)))
 
 (defn- grid-view [{:keys [children rows columns alignment padding spacing]
                    :or {spacing :medium}}]
-  (-> {:fx/type fx.grid-pane/lifecycle}
-      (apply-grid-spacing spacing)
-      (cond->
-        children (assoc :children
-                        (into []
-                              (coll/mapcat-indexed
-                                (fn [row row-children]
-                                  (eduction
-                                    (keep-indexed
-                                      (fn [column child]
-                                        (when child
-                                          (let [{:keys [row_span column_span]
-                                                 :or {row_span 1 column_span 1}} (:props (meta child))]
-                                            (assoc child :grid-pane/row row
-                                                         :grid-pane/column column
-                                                         :grid-pane/row-span row_span
-                                                         :grid-pane/column-span column_span)))))
-                                    row-children)))
-                              children))
-        rows (apply-constraints fx.row-constraints/lifecycle :vgrow rows)
-        columns (apply-constraints fx.column-constraints/lifecycle :hgrow columns)
-        padding (apply-padding padding)
-        alignment (apply-alignment alignment))))
+  (cond->
+    {:fx/type fxui/grid
+     :spacing spacing}
+    children (assoc :children
+                    (into []
+                          (coll/mapcat-indexed
+                            (fn [row row-children]
+                              (eduction
+                                (keep-indexed
+                                  (fn [column child]
+                                    (when child
+                                      (let [{:keys [row_span column_span]
+                                             :or {row_span 1 column_span 1}} (:props (meta child))]
+                                        (assoc child :grid-pane/row row
+                                                     :grid-pane/column column
+                                                     :grid-pane/row-span row_span
+                                                     :grid-pane/column-span column_span)))))
+                                row-children)))
+                          children))
+    rows (apply-constraints :row-constraints fx.row-constraints/lifecycle :vgrow rows)
+    columns (apply-constraints :column-constraints fx.column-constraints/lifecycle :hgrow columns)
```

Comment:
There was actually a bug where using the `rows` prop would instead use the `:column-contraints` key, but with `row-constraint` values, which would cause the runtime exceptions.